build-base/src/main/groovy/readonlyrest.plugin-common-conventions.gradle

```diff
@@ -112,3 +111,36 @@ tasks.register('toJar', Jar) {
     duplicatesStrategy = DuplicatesStrategy.EXCLUDE
     from sourceSets.main.getOutput()
 }
+
+tasks.register('prepareMultiplatformBuilder') {
+    group = 'docker'
+    doLast {
+        def output = new ByteArrayOutputStream()
+        exec {
+            executable = 'docker'
+            args = ['buildx', 'inspect']
+            standardOutput = output
+            errorOutput = System.err
+        }
+        def supportedPlatforms = output.toString().find(/Platforms: ([^\n]+)/)?.replace("Platforms: ", "")?.split(',')
+
+        if (supportedPlatforms == null || !(supportedPlatforms.contains('linux/amd64') && supportedPlatforms.contains('linux/arm64'))) {
+            def rorBuilderExists = exec {
+                executable = 'docker'
+                args = ['buildx', 'inspect', 'ror_kbn_builder']
+            }
+
+            if (rorBuilderExists.exitValue != 0) {
```

Comment:
This line has a problem when the task is run for the first time.

When I run this task manually, the task fails with`ERROR: no builder "ror_kbn_builder" found`
When I run the `localRorDockerImage` task the image builds with success, but with the usage of the wrong builder: 
```
> Task :es816x:localRorDockerImage
#0 building with "desktop-linux" instance using docker driver
```

Reply:
good catch. Fixed